packages/core/src/components/forms/_common.scss

```diff
@@ -151,8 +152,8 @@ $control-group-stack: (
 
 @mixin pt-input-small() {
   height: $pt-input-height-small;
-  padding-right: $pt-grid-size * 0.6;
-  padding-left: $pt-grid-size * 0.6;
+  padding-right: $input-padding-horizontal / 2;
+  padding-left: $input-padding-horizontal / 2;
```

Comment:
this should reference the small padding you just added, not compute a new value.